app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/MultipleDisposalsTriageController.scala

```diff
@@ -1273,7 +1275,7 @@ class MultipleDisposalsTriageController @Inject() (
                 Some(taxYearExchanged),
                 _,
                 _
-              ) if !isAValidCGTTaxTear(taxYearExchanged) =>
+              ) if !isAValidCGTTaxTear(taxYearExchanged, Some(stateYear)) =>
```

Comment:
Use just sting with a new function

Reply:
Same function can be used with Option. But if you want really want another function, I can do that. But thought that it will have same effect without disturbing the others.